### ISSUE TITLE
Update observation activity description

### ISSUE THREAD

**Issue by stscijgbot-rstdms** [spacetelescope/rad#170]
_Issue [RAD-78](https://jira.stsci.edu/browse/RAD-78) was created on JIRA by [David Davis](https://jira.stsci.edu/secure/ViewProfile.jspa?name=ddavis):_

In the observation-1.0.0.yaml file the activity identifier is a 2 digit base 36 number and has a range of 1 - ZZ or 1 - 1295 base 10. Update the title for the archive,

 
```java
  visit_file_activity:
    title: Visit file activity within the sequence, defined range of values is 1..99;
```
to something like
```java
  visit_file_activity:
    title: Visit file activity within the sequence, defined range of values is 1..ZZ; ```
just to make it clear the database should not try and store an integer between 1 and 99.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625115#comment-625115):_

This is implemented in RAD PR-204: #204

**Comment by ddavis-stsci** [comment]
closing since Jira issue is resolved.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625115#comment-625115):_

This is implemented in RAD PR-204: #204

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625115#comment-625115):_

This is implemented in RAD PR-204: #204

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625115#comment-625115):_

This is implemented in RAD PR-204: #204

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625115#comment-625115):_

This is implemented in RAD PR-204: #204

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625774#comment-625774):_

PR approved and merged.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RAD-78?focusedCommentId=625115#comment-625115):_

This is implemented in RAD PR-204: #204
